src/libproc_macro/lib.rs

```diff
@@ -66,6 +66,16 @@ pub struct LexError {
     _inner: (),
 }
 
+#[stable(feature = "proc_macro_lib", since = "1.15.0")]
```

Comment:
```suggestion
#[stable(feature = "proc_macro_lib", since = "1.43.0")]
```
It's certainly preferable to put a correct stabilization version here for documentation, but I'm not sure about the feature name. If it passes `tidy`, then it's good, I guess.

Reply:
So for future reference, next release for stability attributes on new code? (What's the best way to check that? I know how to check the *most recently released* version but not sure how to check what the next upcoming version is.) As for the feature name, I don't imagine that some fairly boilerplate code (which I actually noticed was kinda mentioned as needed in the original commit that added `LexError` in the first place, ecc6c39: "One part of this PR which needs to be improved before stabilizing are the errors and exact interfaces here.") needs its own feature, so it probably is fine as-is, yeah.

Reply:
Oh actually, re: the version, I just looked at the milestones. Why 1.43 and not 1.42?

Reply:
https://forge.rust-lang.org has the schedule for upcoming releases.
`1.43` is because PRs are merged into the Nightly branch, not into Beta.

Reply:
Ah, okay, thank you! This is my first time contributing to Rust so I'm still learning all this stuff :)

Reply:
Okay, so I guess I *will* have to give it its own feature actually? Because the compiler doesn't like just changing the version. ("feature `proc_macro_lib` is declared stable since 1.43.0, but was previously declared stable since 1.15.0")